backend/graph/model/episode.go

```diff
@@ -23,13 +24,25 @@ func EpisodeFromSQL(ctx context.Context, row *sqlc.EpisodeExpanded) *Episode {
 
 	ginCtx, _ := utils.GinCtx(ctx)
 	languages := user.GetLanguagesFromCtx(ginCtx)
+	var season *Season
+	if row.SeasonID.Valid {
+		season = &Season{
+			ID: strconv.Itoa(int(row.SeasonID.Int64)),
+		}
+	}
+
+	var extraDescription string
+	if v := extraDescriptionMap.GetValueOrNil(languages); v != nil {
+		extraDescription = *v
+	}
 
 	episode := &Episode{
 		Chapters:         []*Chapter{}, // Currently not supported
 		ID:               fmt.Sprintf("%d", row.ID),
 		Title:            titleMap.Get(languages),
 		Description:      descriptionMap.Get(languages),
-		ExtraDescription: extraDescriptionMap.Get(languages),
+		ExtraDescription: extraDescription,
+		Season:           season,
```

Comment:
Does the resolver still run if you assign the object here?

Reply:
Yes. Runs as expected, but I could avoid assigning it and instead add SeasonID to Episode, etc, as I only need the ID for the resolvers to work.

Reply:
I think this is perfectly fine then.